Tests/Executor/JobExecutorInlineConfigTest.php

```diff
@@ -322,10 +322,7 @@ public function testTagOverride($storedConfigTag, $requestConfigTag, $requestPar
         ];
 
         $storedConfig = [];
-
-        if ($storedConfigTag !== null) {
-            $storedConfig['runtime']['image_tag'] = $storedConfigTag;
-        }
+        $storedConfig['runtime']['image_tag'] = 'this value will never be used';
```

Comment:
toto tam úplně zapadlo předtím

Reply:
Hmm.. a projevi se to nejak? Ten test prece nijak nekontroluje, ze ta hodnota se potom nikde nepouzije.

Reply:
No neprojeví se to nijak, jen jsem chtěl explicitně zdůraznit to co řešíme tady https://github.com/keboola/docker-bundle/pull/563#discussion_r615848369. Tak testuje se to, že tam je hodnota jiná i když je tahle nastavená na cokoliv.